Documentation/Books/Manual/Downgrading/README.md

```diff
@@ -1,3 +1,38 @@
 Downgrading
 ===========
 
+A direct, binary, downgrade of ArangoDB is **not** supported. If you have upgraded your ArangoDB package, and then also upgraded your current data directory, it is not supported to downgrade the package and start an older ArangoDB version on a data directory that was upgrade already.
```

Comment:
Remove extra comma? `A direct, binary downgrade`

What does _binary_ mean in this context?

Reply:
binary upgrade, or downgrade means in this context: you upgrade or downgrade your binary, then start the server and you do not need to upgrade or restore. this is what we have called "in-place" in https://docs.arangodb.com/devel/Manual/Upgrading/GeneralInfo/  - hope this helps

Reply:
with rocksdb its actually not going to be possible anymore to downgrade to 3.3. 
Startup will just fail when going 3.4 -> 3.3